src/pool/srv_pool.c

```diff
@@ -4735,6 +4780,21 @@ pool_upgrade_props(struct rdb_tx *tx, struct pool_svc *svc,
 		need_commit = true;
 	}
 
+	d_iov_set(&value, &val32, sizeof(val32));
+	rc = rdb_tx_lookup(tx, &svc->ps_root, &ds_pool_prop_reint_mode, &value);
+	if (rc && rc != -DER_NONEXIST) {
+		D_GOTO(out_free, rc);
+	} else if (rc == -DER_NONEXIST) {
+		val32 = DAOS_PROP_PO_REINT_MODE;
```

Comment:
should the value be "DAOS_PROP_PO_REINT_MODE_DEFAULT" rather than the type "DAOS_PROP_PO_REINT_MODE"?

Reply:
Indeed,.